### DISCUSSION TITLE
Is there a way to send HTML as part of message in templateParams

### DISCUSSION THREAD

**Discussion by kavinmehta** [emailjs-com/help#64] [Q&A]
Below is my code to send email: 
`
const templateParams = {
    to_email: toEmail,
    to_name: toName,
    from_name: fromName,
    reply_to: replyTo,
    message,
  }
  emailjs
    .send(
      EMAILJS_SERVICE_ID,
      EMAILJS_TEMPLATE_ID,
      templateParams,
      EMAILJS_USER_ID,
    )
    .then(
      (response) => {
        console.log('SUCCESS!', response.status, response.text)
        return true
      },
      (error) => {
        console.log('FAILED...', error)
        throw error
      },
    )
`
The message attribute is mainly the body of the email.  When I add HTML tags in my 'message' attribute it doesnot render the tags in the email, the tags show up as text e.g. <b></b>. Is there a way to make the message render the HTML tag as well?

![image](https://github.com/emailjs-com/help/assets/142260/d3d30798-afe2-4ea7-8bbd-924556a90fff)
![image](https://github.com/emailjs-com/help/assets/142260/6b044632-8ad9-4fe8-8607-2c8853ca4df4)

**Comment by xr0master** [comment]
You can add HTML code using triple curly braces, but I don't recommend doing this.
Better write a good template with dynamic variables.
